src/containers/PageBuilder/SectionBuilder/SectionBuilder.module.css

```diff
@@ -6,12 +6,29 @@
   display: grid;
   justify-content: start;
   margin: 0 auto;
-  padding: 0 32px 64px;
+  padding: 0 32px;
   position: relative;
 
   @media (--viewportMedium) {
     justify-content: center;
   }
+
+  & + div {
+    padding: 64px 32px 0;
+  }
```

Comment:
Define this before nested media query (and use :only-child)

Reply:
And don't use 3 argument version. Prefer: 2 or 4 for the sake of readability (most ppl/customizers are not familiar with [top] [left&right] [bottom] syntax).

In this case: padding: `64px 32px 0 32px;`